tests/Unit/FeatureTimeBombTest.php

```diff
@@ -24,19 +36,6 @@
        ->toHaveCount(1);
 });
 
-it('Does not throw an Exception when time bombs are disabled', function(): void {
```

Comment:
Sorry, my last comment might have been misleading.  I meant that I found the `$this->assertTrue(true)` assertion unnecessary, not the entire test. I think this test should definitely be kept but maybe use the `tryException` helper you added like in the other tests

Reply:
Ah sorry I get you now :) I've readded this test back in.